src/database.md

```diff
@@ -8,4 +8,14 @@
 
 [ex-sqlite-initialization]: database/sqlite.html#create-a-sqlite-database
 
+## Postgres
```

Comment:
There are 2 types of categories: simple and complex.  Database falls into the simple category, and as such I would prefer to see a single table for the links.  See https://rust-lang-nursery.github.io/rust-cookbook/algorithms.html